README.md

```diff
@@ -31,7 +31,8 @@ This project tries to follow Elasticsearch in terms of [End of Life](https://www
 
 | Elastica                                                                                | ElasticSearch | elasticsearch-php | PHP      |
 | --------------------------------------------------------------------------------------- | ------------- | ----------------- | -------- |
-| [6.x](https://github.com/ruflin/Elastica/tree/master)                                   | 6.x           | ^6.0              | ^7.0     |
+| [7.x](https://github.com/ruflin/Elastica/tree/master)                                   | 7.x           | ^6.0              | ^7.0     |
```

Comment:
I hope we also get a 7.0 version from elasticsearch-php.

Reply:
yep, but not right now, as I learned from them the wait 4 the release of *ES 7.x*
At the moment I had a look and it doesn't seem their working on it. 
But as soon they'll update the library we'll update it.